enumerables.rb

```diff
@@ -0,0 +1,143 @@
+module Enumerable
+  def my_each
+    i = 0
+    while i < length
+      yield (self[i])
+      i += 1
+    end
+  end
+
+  def my_each_with_index
+    i = 0
+    while i < length
+      yield self[i], i
+      i += 1
+    end
+  end
+
+  def my_select
```

Comment:
* [ ] There should be a working `my_select`  identical and returns the same thing as ruby's [`select`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-select). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_select.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
#### Let us compare the results return in each of the examples below and make the return the same result as the original method.
1. ```p [1,2,3,4,5].select``` and ```p [1,2,3,4,5].my_select``` must return same result.
2. ```p (0..5).select{|x| x.even?} ``` and ```p (0..5).my_select{|x| x.even?}``` must return same result.